src/Fixer/PhpUnit/PhpUnitTestAnnotationFixer.php

```diff
@@ -65,6 +65,12 @@ public function testItDoesSomething() {}}'.$this->whitespacesConfig->getLineEndi
         );
     }
 
+    public function getPriority()
+    {
+        // must be run before the PhpdocSeparationFixer and PhpdocOrderFixer
```

Comment:
you need to also add this as a test here:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.8/tests/AutoReview/FixerFactoryTest.php#L182

in tests here, you also have a comments with filename of integration test. you will need to add 2 of them, take a look at other ones so you will get idea how to create them

Reply:
I'm adding those now, as well as running the fixer again